changelog/15524.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:change
+secrets/pki: the signed CA certificate from the /root/sign-intermediate api will now appear within the ca_chain
```

Comment:
Want to elide the `/root` portion, since there's now two of these APIs (`/issuer/:issuer_ref/sign-intermediate` as well)?

Reply:
tweaked.